src/include/miopen/conv/data_invoke_params.hpp

```diff
@@ -38,28 +39,38 @@ struct DataInvokeParams : InvokeParams
     Data_t workSpace;
     std::size_t workSpaceSize;
     bool gfx90aFp16alt;
+    Scalar mAlpha;
```

Comment:
minor: perhaps stick with the naming style of existing members until we decide on a style guide and upgrade the entire class at that point.